javascript/javascript1/week1/script4.js

```diff
@@ -0,0 +1,6 @@
+const firstWords = ["Awesome", "Creative", "Extra-ordinary", "Easy", "Flexible", "All-rounder", "Marvelous", "Integrated", "Embodied", "Corporal"];
+const secondWords = [" Starter", " Launch", " Attempt", " Corporation", " Organization", " Start-up", " Bootup", " Company", " Firm", " Business"];
+
+const randomNumber = Math.floor(Math.random() * 10);
```

Comment:
You're using one random index. Do you think it can better if you used two? So that the combinations of startup names generated is even more random.

Reply:
Yes, that'd be better. Thanks :)